scanner/django.go

```diff
@@ -134,6 +134,39 @@ This module is used on Dockerfile to start the Gunicorn server process.
 		}
 	}
 
+	asgiFiles, err := zglob.Glob(`./**/asgi.py`)
+
+	if err == nil && len(asgiFiles) > 0 {
+		var asgiFilesProject []string
+		for _, asgiPath := range asgiFiles {
+			// When using a virtual environment to manage the dependencies (e.g. venv),
+			// the 'site-packages/' folder is created within the virtual environment
+			// folder. This folder contains all the (dependencies) packages installed
+			// within the virtual environment.
+			// Exclude dependencies matches that contain 'site-packages'.
+			if !strings.Contains(asgiPath, "site-packages") {
+				asgiFilesProject = append(asgiFilesProject, asgiPath)
+			}
+		}
+
+		if len(asgiFilesProject) > 0 {
+			asgiFilesLen := len(asgiFilesProject)
+			dirPath, _ := path.Split(asgiFilesProject[asgiFilesLen-1])
+			dirName := path.Base(dirPath)
+			vars["asgiName"] = dirName
+			vars["asgiFound"] = true
+			if asgiFilesLen > 1 {
+				// Warning: multiple asgi.py files found.
+				s.DeployDocs = s.DeployDocs + fmt.Sprintf(`
+Multiple asgi.py files were found in your Django application:
+[%s]
+Before proceeding, make sure '%s' is the module containing a ASGI application object named 'application'. If not, update your Dockefile.
+This module is used on Dockerfile to start the Gunicorn server process.
```

Comment:
Just a typo here: Daphne instead of Gunicorn.

Reply:
Fixed.